### PR TITLE
demo - some cleanup needed

### PR DESCRIPTION
**Pull request by cpruijsen**

### CODE REVIEW

File: src/client/order-components/GoogleAPI.js [hunk 1]
```diff
@@ -2,7 +2,12 @@
 let map;
 const formatAnswer = (mode, value) => {
   if (!value) {
-    return 'not available';
+    // return 'not available';
+    if (mode === 'fast') {
+      return '5 minutes';
```
Comment (cpruijsen) [inline src/client/order-components/GoogleAPI.js]: what do we want the web client to display if a vendor doesn't return us results?
(improbably in production, but an edge case...) --> `N/A` ok ?

Comment (daredia) [reply src/client/order-components/GoogleAPI.js]: how about `No matching rides available`, though i know thats a bit wordy. N/A looks like an error to me

**Comment by daredia** [comment]
@cpruijsen is this PR ready to merge, or was it just for demo purposes and should be closed?

File: src/client/order-components/GoogleAPI.js [hunk 2]
```diff
@@ -18,14 +23,39 @@ const formatAnswer = (mode, value) => {
   return winnerEstimate;
 };
 
-export const initMap = () => {
+// hacky-global scope variables to manage state for the requestRide
+const rideETA = {};
+const rideFare = {};
+
+export const requestRide = (mode) => {
+  let url = `http://localhost:8000/internal/requestRide`;
```
Comment (daredia) [inline src/client/order-components/GoogleAPI.js:29]: lets make this dynamic based on env variable or else it wont work in prod

File: src/client/order-components/GoogleAPI.js [hunk 3]
```diff
@@ -18,14 +23,39 @@ const formatAnswer = (mode, value) => {
...
+
+export const initMap = (cb) => {
   let origin_place_id = null;
   let destination_place_id = null;
   let travel_mode = 'DRIVING';
   let origin = {};
   let destination = {};
-  let userId = 1; // hardcoded.
-  //let carvisRideId;
+  let userId = 5;
```
Comment (daredia) [inline src/client/order-components/GoogleAPI.js]: should we change this back to 1 to make it more likely to work as a demo?

File: src/client/order-components/GoogleAPI.js (hunk 2 above)
Comment (cpruijsen) [reply src/client/order-components/GoogleAPI.js:29]: created a trello card for this - https://trello.com/c/tnkEnQMS/137-env-node-react not as straight forward as in the server code

File: src/client/order-components/GoogleAPI.js (hunk 3 above)
Comment (cpruijsen) [reply src/client/order-components/GoogleAPI.js]: yeah already done in the current version - dropping DB between each test btw as our env variables were compromised in the travis logs
